test/CMakeLists.txt

```diff
@@ -11,6 +11,13 @@
 # See https://llvm.org/LICENSE.txt for license information.
 #
 ##===----------------------------------------------------------------------===##
+# template_tests
+set (subtract_with_carry_std_template_test.pass_timeout_release "900") # 15min
+set (subtract_with_carry_std_template_test.pass_timeout_debug "700") # 10min
```

Comment:
Since values are quite close (less than 3 minutes difference), do you think it makes sense to implement a special logic to handle the different cases below? I'd consider timeout unification (900 sec) in order to simplify the code below.

Reply:
I would rather keep the separation as a more general implementation case. There is still a chance of having tests with significantly different timeout values.

Reply:
Ok